src/cloud/live/site-launch-checklist.md

```diff
@@ -16,51 +16,80 @@ Before you deploy to the Production environment, download the [Launch checklist]
+
+      We recommend a significantly lower TTL value when switching the DNS record. This value tells the DNS how long to cache the DNS record. When shortened, it refreshes the DNS faster. For example, you can change the TTL value from 3 days to 10 minutes when you are testing your site. Be advised that shortening the TTL value adds load to the web server.
+
+1. Add CNAME records to point the top-level domains and subdomains for your Production environment to the Fastly service `prod.magentocloud.map.fastly.net`, for example:
+
+   | Domain or Subdomain  | CNAME
+   |----------------------|------
+   | www.your-domain.com  | prod.magentocloud.map.fastly.net
+   | mystore.your-domain.com | prod.magentocloud.map.fastly.net
+
+1. If needed, add A records to map the apex domain (`your-domain.com`) to the following Fastly IP addresses:
+   | Apex domain     | ANAME
+   |-----------------|--------
+   | your-domain.com | `151.101.1.124`
+   | your-domain.com | `151.101.65.124`
+   | your-domain.com | `151.101.129.124`
+   | your-domain.com | `151.101.193.124`
+
+1. Update the Magento Base URL.
+
+   -  Use SSH to log in to the Production environment.
+
+      ```bash
+      magento-cloud ssh
+      ```
+
+   -  Use the Magento CLI to change the base url for your store
+
+      ```
+      php bin/magento setup:store-config:set --base-url="https://mcstaging.your-domain.com/"
+      ```
 
-To use this certificate, you must update your DNS configuration so that Magento can complete domain validation and apply the certificate to your environment. Each environment has a unique certificate that covers the domains for the Magento Commerce sites deployed in that environment.
+   {:.bs-callout-info}
+   You can also update the Base URL from the [Magento Admin](https://docs.magento.com/user-guide/stores/store-urls.html#configure-the-base-url).
 
-<!--Note:  The Configure DNS for Fastly service integration topic is coming in later PR-->
+1. Wait a few minutes
 
-See [Configure DNS for Fastly service integration]({{ site.baseurl }}/cloud/cdn/configure-fastly.html).
+1. Test your site.
 
-## Verify Production configurations
+## Verify Production configuration
 
-Make a final pass for any Production configurations in the store(s). If you need to make changes to configurations, you can modify in Production. If settings are read-only, you may need to SSH and CLI commands to modify, or make configuration changes in your local and deploy across.
+Make a final pass to validate the Production configuration for the store(s). If you need to make changes to the configuration, you can modify in Production. If settings are read-only, you may need to open an SSH connection and use CLI commands to modify, or make configuration changes in your local environment. After you complete the updates, you can deploy the changes to Staging and Production environments.
```

Comment:
With `...you can modify in Production.` do you mean modify in the Production environment? not sure, this reads strangely.